### ISSUE TITLE
Bug with ListMany and a single option

### ISSUE THREAD

**Issue by asacolips** [asacolips-projects/pbta#71]
In GitLab by @Asacolips on Apr 29, 2022, 09:22

Reported by `Just Dio is fine` in the Foundry discord:

> When I do a listmany with 2 options it works fine
> ![image](https://gitlab.com/asacolips-projects/foundry-mods/pbta/uploads/ff7f77dd2b1bf4c8d08a5b6167dfe3f8/image.png)
> But if I do a listmany with one option or a checkbox it renders like this for some reason
> ![image](https://gitlab.com/asacolips-projects/foundry-mods/pbta/uploads/f5ebc6142b4bebb7061411cf0ce59025/image.png)
> And its the same if its in the left attributes or top attributes

## Steps to reproduce

```toml
########################################
## NPCS ################################
########################################

[npc]
  # Define attributes.
  [npc.attributesTop]
    [npc.attributesTop.name]
      type = "Text"
      label = "Name"
    [npc.attributesTop.stats]
      type = "Checkbox"
      label = "Stats"
      description = "Check the stats that have been damaged."
  [npc.attributesLeft]
      [npc.attributesLeft.description]
      type = "LongText"
      label = "Description"
  [npc.moveTypes]
      npcMove = "Stat Moves"

#######################################
########### 1 stat ####################
#######################################

[npcOneStat]
    baseType = 'npc'
  # Define attributes.
  [npcOneStat.attributesTop]
    [npcOneStat.attributesTop.name]
      type = "Text"
      label = "Name"
    [npcOneStat.attributesTop.stats]
      type = "ListMany"
      label = "Stats"
      description = "Check the stats that have been damaged."
      options = [". . 1st Stat . ."]
  [npcOneStat.attributesLeft]
      [npcOneStat.attributesLeft.description]
      type = "LongText"
      label = "Description"
  [npcOneStat.moveTypes]
      npcMove = "Stat Moves"

#######################################
########### 2 stats ###################
#######################################

[npcTwoStats]
    baseType = 'npc'
  [npcTwoStats.attributesTop]
    [npcTwoStats.attributesTop.name]
      type = "Text"
      label = "Name"
    [npcTwoStats.attributesTop.stats_2]
      type = "ListMany"
      label = "Stats"
      description = "Check the stats that have been damaged."
      options = [". . 1st Stat . .", ". . 2nd Stat . ."]
  [npcTwoStats.attributesLeft]
    [npcTwoStats.attributesLeft.description]
      type = "LongText"
      label = "Description"
    [npcTwoStats.moveTypes]
      stat = "Stat Moves"
```

## Labels (do-not-change)

/cc @Asacolips

**Comment by asacolips** [comment]
In GitLab by @ActualDio on Apr 30, 2022, 09:21

I also discovered that changing `[npcOneStat.attributesTop.stats]` to `[npcOneStat.attributesTop.stats_1]` fixed the rendering issue for some reason

**Comment by asacolips** [comment]
In GitLab by @Asacolips on May 17, 2022, 20:44

This has been resolved and will be included in the upcoming 0.5.0 release.
